src/components/Apps/UserConfirmDialog/index.vue

```diff
@@ -158,6 +160,7 @@ export default {
       faceToken: null,
       faceCaptureUrl: null,
       noCodeMFA: ['face', 'passkey'],
+      sendCodeMFA: ['email', 'sms'],
       passkeyVisible: false,
       passkeyUrl: '/api/v1/authentication/passkeys/login/?mfa=1'
     }
```

Comment:
The code doesn't contain any regularities, potential issues or significant optimizations. Please ensure that you have the correct inputs and parameters when using them to avoid unexpected errors.